// @widgetbot/embed-api/dist/embed/globalTypes.js
"use strict";
/* tslint:disable */
/* eslint-disable */
// @generated
// This file was automatically generated and should not be edited.
Object.defineProperty(exports, "__esModule", { value: true });
exports.MessageType = exports.MentionType = exports.FormatType = void 0;
//==============================================================
// START Enums and Input Objects
//==============================================================
var FormatType;
(function (FormatType) {
    FormatType["APNG"] = "APNG";
    FormatType["LOTTIE"] = "LOTTIE";
    FormatType["PNG"] = "PNG";
    FormatType["Unknown"] = "Unknown";
})(FormatType = exports.FormatType || (exports.FormatType = {}));
var MentionType;
(function (MentionType) {
    MentionType["Channel"] = "Channel";
    MentionType["Member"] = "Member";
    MentionType["Role"] = "Role";
})(MentionType = exports.MentionType || (exports.MentionType = {}));
var MessageType;
(function (MessageType) {
    MessageType["Call"] = "Call";
    MessageType["ChannelFollowAdd"] = "ChannelFollowAdd";
    MessageType["ChannelIconChange"] = "ChannelIconChange";
    MessageType["ChannelNameChange"] = "ChannelNameChange";
    MessageType["ChannelPinnedMessage"] = "ChannelPinnedMessage";
    MessageType["ChatInputCommand"] = "ChatInputCommand";
    MessageType["ContextMenuCommand"] = "ContextMenuCommand";
    MessageType["Default"] = "Default";
    MessageType["GuildDiscoveryDisqualified"] = "GuildDiscoveryDisqualified";
    MessageType["GuildDiscoveryGracePeriodFinalWarning"] = "GuildDiscoveryGracePeriodFinalWarning";
    MessageType["GuildDiscoveryGracePeriodInitialWarning"] = "GuildDiscoveryGracePeriodInitialWarning";
    MessageType["GuildDiscoveryRequalified"] = "GuildDiscoveryRequalified";
    MessageType["GuildInviteReminder"] = "GuildInviteReminder";
    MessageType["GuildMemberJoin"] = "GuildMemberJoin";
    MessageType["RecipientAdd"] = "RecipientAdd";
    MessageType["RecipientRemove"] = "RecipientRemove";
    MessageType["Reply"] = "Reply";
    MessageType["ThreadCreated"] = "ThreadCreated";
    MessageType["ThreadStarterMessage"] = "ThreadStarterMessage";
    MessageType["Unknown"] = "Unknown";
    MessageType["UserPremiumGuildSubscription"] = "UserPremiumGuildSubscription";
    MessageType["UserPremiumGuildTier1"] = "UserPremiumGuildTier1";
    MessageType["UserPremiumGuildTier2"] = "UserPremiumGuildTier2";
    MessageType["UserPremiumGuildTier3"] = "UserPremiumGuildTier3";
})(MessageType = exports.MessageType || (exports.MessageType = {}));
//==============================================================
// END Enums and Input Objects
//==============================================================
